README.md

```diff
@@ -1,19 +1,19 @@
 # banking 
-[![GoDoc](http://img.shields.io/badge/go-documentation-blue.svg?style=flat-square)](http://godoc.org/github.com/jbub/banking) 
+[![GoDoc](http://img.shields.io/badge/go-documentation-blue.svg?style=flat-square)](http://godoc.org/github.com/kappapay/banking) 
 [![Build Status](https://cloud.drone.io/api/badges/jbub/banking/status.svg)](https://cloud.drone.io/jbub/banking)
-[![Go Report Card](https://goreportcard.com/badge/github.com/jbub/banking)](https://goreportcard.com/report/github.com/jbub/banking)
+[![Go Report Card](https://goreportcard.com/badge/github.com/kappapay/banking)](https://goreportcard.com/report/github.com/kappapay/banking)
 
 Banking library for Go.
```

Comment:
This is a fork of the Banking library for Go. It adds IBAN validation support for [partial IBAN countries](https://www.iban.com/structure).